core/loadpoint.go

```diff
@@ -817,6 +822,9 @@ func (lp *Loadpoint) plannerActive() (active bool) {
 		case requiredDuration < 30*time.Minute:
 			lp.log.DEBUG.Printf("continuing for remaining %v", requiredDuration.Round(time.Second))
 			active = true
+		case lp.clock.Until(planStart) < smallSlotDuration:
+			lp.log.DEBUG.Printf("plan will re-start shortly, continuing for remaining %v", lp.clock.Until(planStart).Round(time.Second))
+			active = true
```

Comment:
Sollten wir nicht auch wenn wir länger eingeschalten lassen als der Planer vorgegeben hat dies speichern, also immer 
```
		lp.setPlanActive(active)
		lp.planSlotEnd = slotEnd
```
machen? Ich hatte jetzt den fall dass zuerst  `plan will re-start shortly, continuing for remaining` und das langsam auf 10 Minuten hochgezählt hat und dann abgeschaltet wurde. (Auto hat schneller geladen als gedacht). Eigentlich hätte ich erwartet dass dann `continuing until end of slot` einspringt, ist aber nicht passiert weil dieser Zustand nicht gespeichert wird. @andig Spricht was dagegend setPlanActive(active) zu setzen vor dem return. Was ist der Grund warum die Änderungen nicht übernommen werden?

Reply:
> @andig Spricht was dagegend setPlanActive(active) zu setzen vor dem return. Was ist der Grund warum die Änderungen nicht übernommen werden?

@schenlap könnten wir leicht im `defer` machen falls active? Nebenwirkungen?

Reply:
OK. Defer war nicht notwendig, ist nur an einer Stelle. Nebenwirkungen: schau ma mal...